src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo-link"
+        href="#"
+      >
+        <img
+          alt="Company logo"
+          class="logo__image"
+          src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1729468800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=CpNPdHLAd0TqrbVi2fnmHIbiEMpLU~AlGpVPmwKvXYnW7wbwafsxiNM914Hmb6ZjkkIgG2AEoQjWEpFsqHdqMNbT1ZvJyALGARF5kopPOVq6chD94e4utrdyVZyDxokK~Jsju8hQHpKETOw42WSd4juAWcLmk4Ms82s9zt717fw~1Kp6ZXJcPqPcf5xA7FYIsaFHC7fVKDRWC5CZu8hE8A-y2noHVVu7DxDKR9Ryz-mUIu0IBIDQwgGOoUSWyzjk~W9gBg3VdZrM1hJ0qYFLZxUyINRWjcT8k-T69WtxO3UbdZKqT6dy71ES97C9Q3UAU5zRiyvu-q1ZttskIC2oow__"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
Add an empty line between multiline sibling blocks of HTML. This will add some 'air' and simplify reading.